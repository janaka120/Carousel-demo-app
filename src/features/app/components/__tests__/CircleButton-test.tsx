import React from 'react';
import renderer from 'react-test-renderer';
import CircleButton from '../CircleButton';

test('CircleButton renders correctly', () => {
  const mockOnPressFn = jest.fn();
  const tree = renderer
    .create(<CircleButton title="<" onPress={mockOnPressFn} disabled={false} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
