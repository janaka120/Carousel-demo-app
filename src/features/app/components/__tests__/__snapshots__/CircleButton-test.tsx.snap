// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleButton renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  nativeID="animatedComponent"
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(117, 208, 227, 0.7)",
      "borderRadius": 35,
      "height": 100,
      "justifyContent": "center",
      "opacity": 1,
      "padding": 0,
      "width": 100,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#FFFFFF",
          "fontSize": 33,
          "lineHeight": 45.375,
        },
        undefined,
      ]
    }
  >
    &lt;
  </Text>
</View>
`;
